pyproject.toml

```diff
@@ -6,6 +6,6 @@ requires = [
     "packaging",
     "py-cpuinfo",
     "Cython >=0.29.21",
-    "blosc2 >=0.6.6"
+    "blosc2 ~=0.6.6"
```

Comment:
See comment below.
Strictly speaking python-blosc2 is not a build requirement, cblosc2 is.